src/main/java/rx/internal/operators/OnSubscribeOnAssembly.java

```diff
@@ -30,7 +31,7 @@
 
     final OnSubscribe<T> source;
     
-    final String stacktrace;
```

Comment:
This field had a purpose to show the stacktrace in debug mode by viewing the field as string!